src/main/java/com/targomo/client/api/quality/criterion/GravitationCriterionDefinition.java

```diff
@@ -6,7 +6,7 @@
 
 import java.util.List;
 
-public interface GravitationCriterionDefinition {
+public interface GravitationCriterionDefinition extends CriterionDefinitionInterface {
```

Comment:
Say, 
'class SomeGravitationCriteria implements GravitationCriterionDefinition  { // implements methods from both the interfaces now} '

but somewhere in the code a valid declaration 
'CriterionDefinitionInterface  someGravitation = new SomeGravitationCriteria()' 

here even though our object "someGravitation" has the method "getTravelType" defined but you cannot call it 
someGravitation.getTravelType

you need this method in the CriterionDefinitionInterface too, probably.